# What is included

Included in this repositroty are:
1) The four simulated datasets used in the paper. 
2) The R code used to produce the three figures in the paper. You will need to change the file_path to match where you have downlaod the data files to. 
3) The three real datasets used in the paper (references to the data are given below)

# Data references

Below are the references for the three real datasets used in the paper. 

E.coli data are derived from:
Goodall, E. C., Robinson, A., Johnston, I. G., Jabbari, S., Turner, K. A.,
Cunningham,A.F.,Lund,P.A.,Cole,J.A.,andHenderson,I.R.(2018).
The essential genome of escherichia coli k-12. MBio, 9(1):e02096–17.

S.aureus data are derived from:
Christiansen, M. T., Kaas, R. S., Chaudhuri, R. R., Holmes,
M. A., Hasman, H., and Aarestrup, F. M. (2014). Genome-wide
high-throughput screening to investigate essential genes involved in
methicillin-resistant staphylococcus aureus sequence type 398 survival.
PLoS One, 9(2):e89018.

S.Typhimurium data are derived from:
Barquist,L.,Boinett,C.J.,andCain,A.K.(2013). Approaches to querying
bacterial genomes with transposon-insertion sequencing. RNA biology,
10(7):1161–1169.

